fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/tree/PhysicalDistributionAggOptRule.java

```diff
@@ -127,9 +141,7 @@ public Void visitPhysicalHashAggregate(OptExpression optExpression, Void context
             PhysicalOlapScanOperator scan = (PhysicalOlapScanOperator) optExpression.getInputs().get(0).getOp();
             PhysicalHashAggregateOperator agg = (PhysicalHashAggregateOperator) optExpression.getOp();
 
-            // Now we only support one-stage AGG
-            // TODO: support multi-stage AGG
-            if (!agg.getType().isGlobal() || agg.getGroupBys().isEmpty()) {
+            if (agg.getGroupBys().isEmpty()) {
```

Comment:
BTW, I think we should add some comment on the class, `PhysicalDistributionAggOptRule` is a meaningless name and nobody known this rule how to work @stdpain

Reply:
PhysicalDistributionAggOptRule means optimize aggregate by physical distribution (hash, sort or colocate)...